src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -115,7 +114,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         {
             $tmp      = tempnam('/tmp', 'dump');
             $contents = file_get_contents($import);
-            $contents = str_replace('#__', 'j_', $contents);
+            $contents = str_replace('#__',  $this->target_db_prefix, $contents);
```

Comment:
@thuebschmann I think you're confusing the *database* prefix with the *Joomla table* prefix here? :) The MySQL database prefix mentioned here is only used for one thing: naming the database.

The `j_` prefix is used for the Joomla *table* names, not the *database*. If you need this one to be changed, we can add a separate option for this, something like `--joomla-table-prefix`. Happy to accept a PR for this.

Reply:
@stevenrombauts 
Yes I think an option for table prefix is the better solution.
So it is posible to install two or more joomla in one database. 
Can you add this obtion?